### ISSUE TITLE
Add a license for game/renderware_binary_stream.ksy

### ISSUE THREAD

**Issue by generalmimon** [kaitai-io/kaitai_struct_formats#412]
The spec [`game/renderware_binary_stream.ksy`](https://github.com/kaitai-io/kaitai_struct_formats/blob/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy) has no `/meta/license` key, which means that it is under exclusive copyright. All contributors hold the copyright to the lines of code that they contributed and nobody else shall legally use them (see https://choosealicense.com/no-permission/).

Therefore, it is desirable to choose a license for it. I'll put a list of contributors here according to [Git history](https://github.com/kaitai-io/kaitai_struct_formats/commits/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy). If you're mentioned, **please comment whether you are OK with releasing the [`renderware_binary_stream.ksy`](https://github.com/kaitai-io/kaitai_struct_formats/blob/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy) spec under the [`CC0-1.0` license](https://spdx.org/licenses/CC0-1.0.html)**. _(If not, suggest an alternative from [this list](https://spdx.org/licenses/).)_

* [`renderware_binary_stream.ksy`](https://github.com/kaitai-io/kaitai_struct_formats/commits/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy)

  - [ ] @mnakamura1337 ([commits](https://github.com/kaitai-io/kaitai_struct_formats/commits/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy?author=mnakamura1337))
  - [ ] @Ithamar ([commits](https://github.com/kaitai-io/kaitai_struct_formats/commits/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy?author=Ithamar))
  - [ ] @GreyCat ([commits](https://github.com/kaitai-io/kaitai_struct_formats/commits/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy?author=GreyCat))
  - [ ] @generalmimon ([commits](https://github.com/kaitai-io/kaitai_struct_formats/commits/1eae5145325bb992e20dae838dd68d04814dda46/game/renderware_binary_stream.ksy?author=generalmimon))

**Comment by Ithamar** [comment]
I agree with the relicensing of my changes to CC0-1.0. If you need any more, just let me know.

**Comment by GreyCat** [comment]
I hereby agree with licensing my changes as CC0-1.0.

**Comment by generalmimon** [comment]
I'm also OK with releasing my changes under `CC0-1.0`.

However, I doubt that we'll ever be able to reach out to @mnakamura1337, who is the original author. He seems to be missing in action since 2017 - his latest activity at both [GitHub](https://github.com/mnakamura1337?tab=overview&from=2017-12-01&to=2017-12-31) and [Habr](https://habr.com/ru/users/mnakamura/) was on July 17, 2017 and then we know nothing about him. He has no GitHub visits/contributions for 2018-2021. I sent him an e-mail on 8 Feb, 2021, but his email box is apparently canceled:

> This message was created automatically by mail delivery software.
> A message that you sent could not be delivered to one or more of its recipients. This is a permanent error.
> 
> mnakamura1337@yahoo.co.jp, ERROR CODE :554 - delivery error: dd Sorry your message to mnakamura1337@yahoo.co.jp cannot be delivered. This account has been disabled or discontinued [#102]. - mta114.mail.bbt.yahoo.co.jp

So we're pretty much screwed - we can't apply any license without his consent, which we're unlikely to get.
